[Deploy] Revert "[Deploy] Backend Server CI/CD 구축"

miraclee1226 opened the pull request:
변경사항
- pnpm install version 8.15.6로 수정

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Chores**
	- GitHub Actions 워크플로우에서 `pnpm` 버전을 `8.15.6`으로 업데이트
	- CloudFront 및 Express 배포 워크플로우의 `pnpm` 설치 단계 조정
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

miraclee1226 committed:
Revert "[Deploy] Backend Server CI/CD 구축 (#172)"

This reverts commit a5202ad99419121f393cfc2a98f877925353bd57.

miraclee1226 committed:
Fix: packageManager 버전 수정

miraclee1226 committed:
Deploy: deploy-express.yml 생성

bokeeeey commented on package.json:
```diff
@@ -22,7 +22,7 @@
     "typescript": "^5.6.3"
   },
-  "packageManager": "pnpm@8.15.6",
+  "packageManager": "pnpm@9.12.1",
```
나도 packageManager 버젼은 안건드리는게 좋을거같은데 메이저 버젼이 업데이트되면서 발생할 변수가 좀 있을거같아

bokeeeey commented on deploy.sh:
이거 ec2에서 테스트는 해본거야?

bokeeeey replied:
명령어 아마 좀 다를거야 기존이랑

miraclee1226 committed:
Remove: deploy.sh 삭제

miraclee1226 committed:
Chore: packageManager 버전 수정정

miraclee1226 committed:
Fix: pnpm 버전을 package.json과 일치하도록 수정

miraclee1226 committed:
Refactor: deploy-cloudfront.yml 최적화

- Actions 버전 업데이트 (checkout, setup-node, pnpm-action)
- Node.js 20.x 버전으로 업그레이드
- pnpm 캐싱 추가
- --frozen-lockfile 플래그 추가
